pyro/contrib/gp/models/sgpr.py

```diff
@@ -112,12 +109,10 @@ def __init__(self, X, y, kernel, Xu, noise=None, mean_function=None, approx=None
             raise ValueError("The sparse approximation method should be one of "
                              "'DTC', 'FITC', 'VFE'.")
 
+    @autoname.scope(prefix="SGPR")
```

Comment:
Creating a scope is in unnecessary now. It is just useful when we want to combine many gp models.